src/ws/cockpitchannelresponse.c

```diff
@@ -26,9 +26,12 @@
 
 #include <string.h>
 
+#define CHECKSUM_HEADER "X-Cockpit-Pkg-Checksum"
```

Comment:
Just using etag means parsing out the language from the etag. I'd rather be explicit that's why I added a custom header. Agree?